vms/platformvm/vm_test.go

```diff
@@ -2139,14 +2105,14 @@ func TestRemovePermissionedValidatorDuringAddPending(t *testing.T) {
 	nodeID := ids.GenerateTestNodeID()
 
 	addValidatorTx, err := vm.txBuilder.NewAddValidatorTx(
-		defaultMaxValidatorStake,
+		ts.MinValidatorStake,
```

Comment:
`ts.Keys[0]` is used to fund a subnet so it has not enough balance to create a validator with MaxStake. This was not a problem before because fork time was not properly set and fee for subnet creation was zero